components/clp-package-utils/clp_package_utils/scripts/decompress.py

```diff
@@ -76,6 +75,13 @@ def main(argv):
         return -1
     extraction_dir.mkdir(exist_ok=True)
```

Comment:
Can we move this after the config file validation so that there's fewer (or no) side-effects on failure?